src/frontends/pytorch/src/op_table.cpp

```diff
@@ -304,7 +305,8 @@ OP_CONVERTER(translate_new_ones_fx);
 OP_CONVERTER(translate_new_zeros_fx);
 OP_CONVERTER(translate_ones_fx);
 OP_CONVERTER(translate_ones_like_fx);
-OP_CONVERTER(translate_reflection_pad_nd_fx);
+OP_CONVERTER(translate_reflection_pad_nd);
+OP_CONVERTER(translate_reshape_fx);
```

Comment:
Why do you add reshape here?

Reply:
I think some changes from my other local experiment branches mistakenly persisted.
Thank you for pointing it out.